src/develop/components/ColumnDoc.js

```diff
@@ -0,0 +1,157 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+var React = require('react');
+var jsxToString = require('jsx-to-string');
+var DocsArticle = require('../../DocsArticle');
+var OneColumn = require('../../modules/OneColumn');
+var TwoColumn = require('../../modules/TwoColumn');
+var ColumnContent = require('../../modules/ColumnContent');
+var DownloadIcon = require('grommet/components/icons/base/Download');
+
+OneColumn.displayName = 'OneColumn';
+
+var inline = "<{Number}Column />";
```

Comment:
Can we change this example snippet of code to something more descriptive like this instead:

```
<OneColumn headline="Headline Title" />
<TwoColumns>
  <ColumnContent title="Title" />
</TwoColumns>
```

I was a little confused about what {Number} could be.

With that said, do you think it would be better to break this doc page into two, one for just OneColumn and the other for TwoColumns + ColumnContent?